src/main/java/frc/robot/commands/MoveToSelectedPose.java

```diff
@@ -0,0 +1,40 @@
+package frc.robot.commands;
```

Comment:
This whole class could prorobly a subclass of 





























this command should probobly be a subclass of the GoToPoseCommand

Reply:
There is no GoToPose command on this branch. I think it's on an unmerged branch.